src/main/java/org/motechproject/ebodac/util/DtoLookupHelper.java

```diff
@@ -47,12 +49,14 @@ public static GridSettings changeLookupForFollowupsAfterPrimeInjectionReport(Gri
         return settings;
     }
 
-    public static GridSettings changeLookupAndOrderForFollowupsMissedClinicVisitsReport(GridSettings settings) {
+    public static GridSettings changeLookupAndOrderForFollowupsMissedClinicVisitsReport(GridSettings settings) throws IOException {
         DateTimeFormatter formatter = DateTimeFormat.forPattern("yyyy-MM-dd");
+        Map<String,String> fieldsMap = new HashMap<>();
+
         if (StringUtils.isNotBlank(settings.getSortColumn())) {
             String sortColumn = settings.getSortColumn();
             if (sortColumn.equals("planedVisitDate") || sortColumn.equals("noOfDaysExceededVisit")) {
-                settings.setSortColumn("motechProjectedDate");
+                settings.setSortColumn(Visit.MOTECH_PROJECTED_DATE_PROPERTY_NAME);
                 if(settings.getSortDirection().equals("asc")) {
```

Comment:
sort direction should not be changed for planedVisitDate only for noOfDaysExceededVisit